examples/with_winit/src/main.rs

```diff
@@ -49,10 +53,51 @@ struct Args {
     /// Switch between scenes with left and right arrow keys
     #[arg(long)]
     scene: Option<i32>,
+    #[arg(
+        long,
+        default_value_t = ClearColor::Black,
+        value_parser = PossibleValuesParser::new(["black", "white", "aquamarine", "crimson"])
+            .map(|s| s.parse::<ClearColor>().unwrap())
+    )]
+    clear_color: ClearColor,
     #[command(flatten)]
     args: scenes::Arguments,
 }
 
+#[derive(Debug, Clone)]
+enum ClearColor {
```

Comment:
The peniko `Color` type has a [parse](https://github.com/linebender/peniko/blob/9526fb1afa85493cf537afac76c1e38dc7c76d23/src/color.rs#L128) method that supports hex values and svg color names. Can we make use of that here instead of a new enum with a limited set of colors?

Reply:
Great! Done.